packages/react-components/src/styles/index.ts

```diff
@@ -202,7 +203,7 @@ export default createGlobalStyle<Props & ThemeProps>(({ theme, uiHighlight }: Pr
     .ui--Toggle.isChecked {
       &:not(.isRadio) {
         .ui--Toggle-Slider {
-          background-color: ${getHighlight(uiHighlight)} !important;
+          background-color: ${getHighlight(uiAltHighlight)} !important;
```

Comment:
This can simply be `background: ${getHighlight(uiHighlight)} !important;` with non of the alt colors.

Reply:
cool, will fix it!